### PR TITLE
[MBL-15913][Student][Teacher] Discussion Redesign, anonymous replies don't load

### PR DESCRIPTION
**Pull request by tamaskozmer**
Test plan: 
- Create an anonymous discussion with the discussions redesign (one is included in the ticket)
- Open the anonymous discussion
- You should see a text that the discussion is not supported on mobile and a link to open the discussion in a WebView.
- Verify that the discussion opens
- Also smoke test regular discussions

refs: MBL-15913
affects: Teacher, Student
release note: Added a button to open anonymous discussions in browser.

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

File: apps/teacher/src/main/java/com/instructure/teacher/fragments/DiscussionsDetailsFragment.kt [hunk 1]
```diff
@@ -713,12 +705,24 @@ class DiscussionsDetailsFragment : BasePresenterFragment<
...
 
+    override fun showAnonymousDiscussionView() {
+        anonymousDiscussionsNotSupported.setVisible()
+        openInBrowser.setVisible()
+        replyToDiscussionTopic.setGone()
+        swipeRefreshLayout.isEnabled = false
+        openInBrowser.onClick {
+            mDiscussionTopicHeader.htmlUrl?.let { url ->
+                requireContext().startActivity(InternalWebViewActivity.createIntent(requireContext(), url, "", true))
+            }
```
Comment (hermannakos) [inline apps/teacher/src/main/java/com/instructure/teacher/fragments/DiscussionsDetailsFragment.kt:716]: Maybe we should hide the button if for some reason the url is not available or have some error handling.

**Review by hermannakos — CHANGES_REQUESTED** [review]

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by kdeakinstructure — APPROVED** [review]
QA 👍

**Pull request merged**